Libplanet/Assets/Currency.cs

```diff
@@ -311,7 +361,7 @@ private Currency(SerializationInfo info, StreamingContext context)
         private Currency(
```

Comment:
This private constructor was originally intended only for `Currency.Capped()` factory method, so if it is meant to also accompany `Currency.Uncapped()`, I guess you might have wanted to modify the `<summary />` for the constructor as well. 😅

Reply:
Adjusted the XML comment!